services/fxa/env.yml

```diff
@@ -13,14 +13,10 @@ default:
   VERSION: ${env:VERSION}
   REMOTE_ORIGIN_URL: ${env:REMOTE_ORIGIN_URL}
   LOG_LEVEL: ${env:LOG_LEVEL}
-  NEW_RELIC_ACCOUNT_ID: ${env:NEW_RELIC_ACCOUNT_ID}
```

Comment:
These remenants of New Relic were found during the process of removing AWS XRay support.